pkg/network/protocols/postgres/protocol.go

```diff
@@ -230,3 +253,44 @@ func (p *protocol) setupMapCleaner(mgr *manager.Manager) {
 
 	p.mapCleaner = mapCleaner
 }
+
+func (p *protocol) updatePgKernelTelemetry(mgr *manager.Manager) {
+	mapPlain, err := protocols.GetMap(mgr, KernelTelemetryMapPlain)
+	if err != nil {
+		log.Warn(err)
```

Comment:
why not error?

Reply:
this is what http2 protocol does, I can change it to error.